_posts/2024-03-19-FOSSRIT-bug-fix.md

```diff
@@ -0,0 +1,47 @@
+
+
+My Comm Arch experiences did not cover this project, however what we covered in the three presentations showed me that the FOSSRIT website would be good to work on. Before Adrien and I started contributing to the project again, the previous commit was from October 2022, where Adrien and others were contributing to the site in their free time. This website is more of a student-led hobby project with the support of the faculty interested in Open Source at RIT, but the site needed some changes after not being updated in two years.
+
+
+As the only active maintainer at the time, Adrien has been open to answering any questions I have and giving input when applicable to my changes. We talked about how Jekyll (which is how to build the site locally) works and what could be added/changed on the site.
+
+
+## The issue
+To decide which issue I was going to tackle, I visited the github issues page while talking to Adrien and I just read what the issues were trying to resolve. I was interested in [Issue #188](https://github.com/FOSSRIT/fossrit.github.io/issues/188) of the project, which was titled _"point open at RIT page to the current open@RIT site"_.
+
+
+I first went about pursuing the issue by changing the contents of the [Open@RIT page](https://fossrit.github.io/open-rit/) on the website, however Adrien liked the snippets that were talking about Open@RIT's history, so I left that for now... (That is my plan for my contribution!). The issue was that the Open@RIT page was really a placeholder until the official [Open@RIT](https://openr.it/) page was created! There is a history as to why Open@RIT has its own separate website from RIT, which I won't be covering in my contribution content but the plan is to cover that topic in the slides.
+
+
+![The Pull Request](https://i.imgur.com/Z8zpQn4.png)
+Once I completed my changes, I made my [pull request](https://github.com/FOSSRIT/fossrit.github.io/pull/190) to the parent repository. In this pull request, Adrien and I go back and forth about what the changes I make and potential revisions and ideas that sprout from the fix. I had to go back and revise quite a few times so I kept on committing to the pull request, which wasn't intentional after all the issues were fixed.
+
+
+![Some small minor details](https://i.imgur.com/cvdQAxw.png)
+What blocked me at first in getting my pull request approved was the small minor details, such as the history that Open@RIT was accepted by the provost in 2020. I removed much more content than I should have as well, so a lot of the work inside the pull request was reintroducing that information. Once I completed those changes, I had succeeded getting my pull request merged! (And came up with a great idea for my contribution!)
+
+
+The bug fix also added documentation on the best practice for how to contribute to the github project, while the contribution will focus on how to locally build the website along with the history update.
+
+
+## Contribution
```

Comment:
you probably dont *need* to have this here since the contribution is a separate blog post

Reply:
Fair enough! Will remove contribution, was just going down the rubric.